vlsi/Makefile

```diff
@@ -99,6 +99,8 @@ SIM_TIMING_CONF = $(OBJ_DIR)/sim-timing-inputs.yml
 include $(vlsi_dir)/sim.mk
 $(SIM_CONF): $(VLSI_RTL) $(HARNESS_FILE) $(HARNESS_SMEMS_FILE) $(sim_common_files) $(dramsim_lib)
 	mkdir -p $(dir $@)
+	mkdir -p $(OBJ_DIR)/$(HAMMER_SIM_RUN_DIR)/$(notdir $(BINARY))
+	ln -sf $(base_dir)/generators/testchipip/src/main/resources/dramsim2_ini $(OBJ_DIR)/$(HAMMER_SIM_RUN_DIR)/$(notdir $(BINARY))/dramsim2_ini
```

Comment:
is this not set as a resource with `setResource`? It feels like that would solve this automatically, but maybe you'd still need another `ln -s`

Reply:
Currently the dramsim blackbox in testchipip only looks in the current directory and since hammer runs the sims from a different directory based on all these factors we can't just commit the symlink like `sims/vcs` does.